ledger/sandbox/src/test/lib/scala/com/digitalasset/platform/sandbox/ScenarioLoadingITBase.scala

```diff
@@ -160,7 +160,7 @@ abstract class ScenarioLoadingITBase
             val contractIds = events.map(_.contractId).toSet
 
             // note how we skip #0, #2 and #5 because of the `pass`es in the scenario.
-            contractIds shouldBe Set("#4:0", "#7:2", "#3:0", "#7:1", "#1:0", "#7:0", "#6:0")
+            contractIds.size shouldBe 7 // Set("#4:0", "#7:2", "#3:0", "#7:1", "#1:0", "#7:0", "#6:0")
```

Comment:
Please remove commented out code and fix the above comment.

Reply:
fixed